earth_enterprise/src/fusion/gepublish/geserveradmin.cpp

```diff
@@ -93,6 +93,10 @@ void usage(const std::string &progn, const char *msg = 0, ...) {
     "   [--vhname <vh_name>]     If virtual host is omitted,\n"
     "                            <db_name> is published to the default\n"
     "                            virtual host: \"%s\".\n"
+    "   [--setecdefault]         Publish this database as the default one for\n"
```

Comment:
We really ought to update the docs for this as well, which is going to be a problem since we haven't updated the docs for 5.2.2 yet. We'll have to do the task first to create docs for 5.2.2, so this might be better handled as a follow up.

Docs that need updating:
- http://www.opengee.org/geedocs/answer/3481558.html (command line reference)
- http://www.opengee.org/geedocs/answer/6034724.html (may need to update some images)
- We could also add a new documentation page or a paragraph somewhere explaining the default globe capability